taxi_zone_lookup.csv

```diff
@@ -0,0 +1,266 @@
+"LocationID","Borough","Zone","service_zone"

```

Comment:
This file should not be committed to the repository. Try to "untrack" this file in git and add this fil to `.gitignore`